### DISCUSSION TITLE
Issues exporting to Flight Simulator with SU10 beta

### DISCUSSION THREAD

**Discussion by erdebugger** [albar965/littlenavmap#930] [General]
[VFR Skukuza (FASZ) to Selebi-Phikwe (FBSP).zip](https://github.com/albar965/littlenavmap/files/9577931/VFR.Skukuza.FASZ.to.Selebi-Phikwe.FBSP.zip)
![pic3](https://user-images.githubusercontent.com/17746363/190480776-32036f46-033a-4f94-ab44-416237d4ee19.png)
![pic2](https://user-images.githubusercontent.com/17746363/190480781-cb4555ed-e81c-4592-ab92-0144c005db3e.png)
![pic1](https://user-images.githubusercontent.com/17746363/190480782-06327fb8-0ba1-4ac4-9799-b1f8217bd5d4.png)

I am attaching a sample flight plan, which looks great in LittleNavmap, but when exported to Flight Simulator, it produces points at coordinate 0,0. If I have the flight plan loaded by the Garmin pms50-gns530, by copying the .pln file to the /fpl530 folder, it only recognizes the origin and destination airports, but does not recognize any intermediate point. I am also attaching an image of how the flight plan appears when it is loaded into Flight Simulator from the world map screen.

**Comment by albar965** [comment]
Give me a day or two. Looking at other issues now.

**Comment by albar965** [comment]
MSFS issue. Attached plan was created using MSFS and it still shows the same issues when loading. So, MSFS cannot load its own plans.
I'd advise to report this to Asobo.

[EDDL EDDS Custom WPT MSFS.PLN.txt](https://github.com/albar965/littlenavmap/files/9584574/EDDL.EDDS.Custom.WPT.MSFS.PLN.txt)
